packages/webview_flutter/webview_flutter_wkwebview/ios/Classes/FWFDataConverters.m

```diff
@@ -192,9 +192,16 @@ WKNavigationActionPolicy FWFNativeWKNavigationActionPolicyFromEnumData(
 }
 
 FWFNSErrorData *FWFNSErrorDataFromNativeNSError(NSError *error) {
-  return [FWFNSErrorData makeWithCode:@(error.code)
-                               domain:error.domain
-                 localizedDescription:error.localizedDescription];
+  NSMutableDictionary *userInfo;
+  if (error.userInfo) {
+    userInfo = [NSMutableDictionary dictionary];
+    for (NSErrorUserInfoKey key in error.userInfo.allKeys) {
+      if ([error.userInfo[key] isKindOfClass:[NSString class]]) {
+        userInfo[key] = error.userInfo[key];
+      }
+    }
```

Comment:
The value can be any object from any library, so this only checks for Strings. This could also check for the other pigeon supported values too, e.g. `NSNumbers` too if needed. We currently only use the failingURL and localizedDescription.